database_updates.php

```diff
@@ -1132,7 +1132,12 @@
         mysqli_query($mysqli, "UPDATE `settings` SET `config_current_database_version` = '0.6.1'");
     }
 
-    //if (CURRENT_DATABASE_VERSION == '0.6.1') {
+    if (CURRENT_DATABASE_VERSION == '0.6.1') {
+        mysqli_query($mysqli, "ALTER TABLE `settings` ADD COLUMN `config_imap_username` VARCHAR(200) NULL DEFAULT NULL AFTER `config_imap_encryption`");
+        mysqli_query($mysqli, "ALTER TABLE `settings` ADD COLUMN `config_imap_password` VARCHAR(200) NULL DEFAULT NULL AFTER `config_imap_username`;");
```

Comment:
- Remove semi colon at end of query statement.

- Add update statement for database version